sessionize/lib/src/commonMain/kotlin/co/touchlab/sessionize/AppContext.kt

```diff
@@ -155,6 +170,26 @@ object AppContext {
     private fun storeAll(networkSponsorJson: String, networkSpeakerJson: String, networkSessionJson: String) {
         dbHelper.primeAll(networkSpeakerJson, networkSessionJson, networkSponsorJson)
     }
+
+    private fun createNotificationsForSessions() = coroutineScope.lateValue.launch {
+
+        val mySessions = backgroundSuspend {
+            sessionQueries.mySessions().executeAsList()
+        }
+
+        for (sess:MySessions in mySessions) {
```

Comment:
mySessions and sessionById are dealing with the same table. Seems like it would make sense to update the mySessions query to return the columns needed here instead of doing a lot of extra queries.

Reply:
Right now the notification includes the room name, should this be kept in? This would require a join between two tables

Reply:
done in latest commit

Reply:
I'm confused why it would need a join when you didn't need one before. Might be me not understanding sqldelight, but the query for sessionById appears to be "SELECT * FROM session WHERE id = ?1" and doesn't mention the room. I'm not sure where the `session.room()` call gets generated, assume its essentially making the query, does that not work when you get the list?

Reply:
Tried this out locally, and I see now that the room() call is an extension we define which makes the query. I am wondering if joining it makes more sense to join it than to make the call down the road. I see we do something similar for the schedule view

Reply:
As we discussed earlier, this wouldnt work well in the ScheduleModel because it's constantly being called and would be too repetitive.